experiment.py

```diff
@@ -44,25 +50,37 @@ def Scan_Field(stop):
         client.field.approach_mode.linear,
         client.field.driven_mode.driven
     )
-    
- 
+
+    # setup Keithley
+    keithley = Keithley2400("GPIB0::5")
+
+    keithley.apply_voltage()
+    keithley.compliance_current = 0.1
+    keithley.enable_source()
+
     for t in range(2501):
-        # chamber conditions
-        T, F, C, res = save_temp_field_chamber()
-        
-        data.set_value('Time', t)
-        
-        data.set_value('Temperature', T)
-        
-        data.set_value('Field', F)
-        
-        data.set_value('Chamber Status', C)
-        
-        data.set_value('Resistance', res)
-        data.write_data()
-        
-        # poll data at roughly equal intervals based on points/ramp
-        time.sleep(wait)
+
+        # Keithley sweep
+        v_list = np.linspace(0, -16, 1)  # same range as Sammak et. al.
+
+        for v in v_list:
+            keithley.ramp_to_voltage(v, steps=10, pause=0.001)  # ramp Keithley very quickly
+            print(f'Vg: {keithley.voltage} ')
+
+            # chamber conditions
+            T, F, C, res = save_temp_field_chamber()
+
+            data.set_value('Time', t)
+            data.set_value('Temperature', T)
+            data.set_value('Field', F)
+            data.set_value('Chamber Status', C)
+            data.set_value('Resistance', res)
+            data.write_data()
+
+            # poll data at roughly equal intervals based on points/ramp
+            time.sleep(wait)

```

Comment:
take time.sleep(wait) outside the loop
maybe we want to substract the time ramp_to_voltage takes in time.sleep(wait)